fetchquotes/fetchquotes.test.js

```diff
@@ -0,0 +1,9 @@
+import fetchQuote from './fetchquotes';
+
+describe('fetch futurama quote', () => {
+  it('should fetch and return a single quote from the futurama API in specific format', async () => {
+    const characterQuote = await fetchQuote();
+
+    expect.objectContaining(characterQuote);
```

Comment:
The `expect` statement here needs to be fleshed out - right now it's not really comparing anything, but you're on the right track.

You can compare like this:
```js
expect(characterQuote).toEqual(expect.objectContaining({
character: expect.any(String),
text: expect.any(String),
image: expect.any(String)
}));
```